Formula/navi.rb

```diff
@@ -11,14 +11,35 @@ class Navi < Formula
     sha256 "64b6dac7d4c689ab6aa5c88ef58369249af2d9a6ac44f56769685569a673dd56" => :high_sierra
   end
 
+  depends_on "rust" => :build
   depends_on "fzf"
 
   def install
-    libexec.install Dir["*"]
-    bin.write_exec_script(libexec/"navi")
+    # copy resources
```

Comment:
No need to comment, people familiar with brew/ruby/packaging can just read the code.